src/test/harness.js

```diff
@@ -548,9 +548,11 @@ async function toggleScopeNode(text) {
   return toggleObjectInspectorNode(node);
 }
 
-function writeInConsole(value) {
+async function writeInConsole(value) {
   window.jsterm.setValue(value);
-  return waitUntil(() => window.jsterm.editor.getValue() === value);
+  await waitUntil(() => window.jsterm.editor.getValue() === value);
+  // workaround for #6774
```

Comment:
Can we add a TODO comment here and pin it to a task? This (and the code it's working around) should be tracked.